### ISSUE TITLE
Tight bond cards do not work correctly when played in sets of three

### ISSUE THREAD

**Issue by reem** [exane/gwent#3]
The middle of the three cards has its value increased x4, as it should, but the outer two are only doubled. In the game all three are increased x4.

**Comment by supermax313** [comment]
Yes, placing this unit next to a card with the same name will double
the strength of both card. 2 cards => x2, 3 cards => x3. Why x4?
1 card => ability = off
2 card => both strength x2
3 card => all strength x3
More than 3 cards with the same name with this skill in the deck can not be.

**Comment by exane** [comment]
My previous thought about tight bonding was a bit off. I read it a bit to literally. I changed it and now it should work as supermax313 suggested.

**Comment by supermax313** [comment]
I am now in Witcher 3 will check the situation.
